examples/create-react-app/.env

```diff
@@ -0,0 +1,3 @@
+SASS_PATH=node_modules:src
+SKIP_PREFLIGHT_CHECK=true
+EXTEND_ESLINT=true
```

Comment:
Same with this, we can leave comments here explaining this allows you to override create react app default linting